plugins/keyboard-navigation/test/navigation_modify_test.mocha.js

```diff
@@ -8,27 +8,10 @@ const chai = require('chai');
 const Blockly = require('blockly/node');
 const {Navigation} = require('../src/navigation');
 const assert = chai.assert;
+const { testHelpers } = require('@blockly/dev-tools');
+const { captureWarnings } = testHelpers;
 
 suite('Insert/Modify', function() {
-  /**
-   * Captures the strings sent to console.warn() when calling a function.
-   * Copies from core.
-   * @param {function} innerFunc The function where warnings may called.
-   * @return {string[]} The warning messages (only the first arguments).
-   */
-  function captureWarnings(innerFunc) {
-    const msgs = [];
-    const nativeConsoleWarn = console.warn;
-    try {
-      console.warn = function(msg) {
-        msgs.push(msg);
-      };
-      innerFunc();
-    } finally {
-      console.warn = nativeConsoleWarn;
-    }
-    return msgs;
-  }
 
```

Comment:
The eslint check ("Block must not be padded by blank lines") is complaining about this extra line here.